### DISCUSSION TITLE
Precracked test sample

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#296] [Q&A] [answered]
Dear Peridigm community,

I am working on test/regression/precracked sample, but when I am running the code this is how my output is showing in Paraview,

Is this correct or because of some error it is showing like this?

![Screenshot (64)](https://user-images.githubusercontent.com/110670051/215506943-611bf350-d20a-4776-a72c-20ebb4c73bc7.png)

**Comment by djlittl** [comment, marked as answer]
Seems reasonable, looks like the precrack is a horizontal plane on the left-hand side, midway up in the vertical direction.  It's a bit hard to see because the points are small in ParaView (there are several ways to make them bigger).  A more convincing way to visualize would be to show the sample under load and plot displacement in the y direction.  You should see the crack opening up.

**Reply by Nikhil23121998** [reply]
Ok, Thankyou.
